templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -303,5 +402,37 @@
         cj('#record-different-contact').hide();
       }
     }
+
+    function buildAmount( ) {
```

Comment:
I'd feel much better if this was consolidated into a function shared with the membership.tpl form